src/app/page.tsx

```diff
@@ -1,101 +1,78 @@
-import Image from "next/image";
+import React from 'react';
+import { Lock, User, CheckCircle, Shield } from 'lucide-react';
 
-export default function Home() {
+const SecureAuthenticationUI = () => {
```

Comment:
If this is a reusable component, it should go to `src/components` Lets stick to Nextjs convention of Pages. Please see page documentation. https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts